packages/flame_riverpod/pubspec.yaml

```
@@ -0,0 +1,18 @@
+name: flame_riverpod
+description: Helpers for using Riverpod - a reactive caching and data-binding framework, 
+  in conjunction with Flame. The ComponentRef is used as an intermediary between Flame and
+  the Widget tree in order to provide state to Flame Components.
+version: 4.0.0+2
+homepage: https://github.com/flame-engine/flame/tree/main/packages/flame_riverpod
+environment:
+  sdk: '>=2.18.5 <3.0.0'
```

nit:
```suggestion
  sdk: ">=2.18.5 <3.0.0"
```
(see https://github.com/flame-engine/flame/pull/2374)